### PR TITLE
[ticket/13672] Allow changing the data from the user-submitted report

### PR DESCRIPTION
**Pull request by brunoais**
https://tracker.phpbb.com/browse/PHPBB3-13672

### CODE REVIEW

File: phpBB/includes/mcp/mcp_reports.php [hunk 1]
```diff
@@ -73,18 +73,47 @@ function main($id, $mode)
...
+				/**
+				* Allow changing the data obtained from the user-submitted report.
+				*
+				* @event core.mcp_reports_report_details_query_after
+				* @var	array	sql_ary		The array in the format of the query builder with the query that had been executted
+				* @var	mixed	forum_id	The forum_id, the number in the f GET parameter
+				* @var	int		post_id		The post_id of the report being viewed (if 0, it is meaningless)
+				* @var	int		report_id	The report_id of the report being viewed
+				* @var	int		report		The query's resulting row.
+				* @since 3.1.4-RC1
```
Comment (marc1706) [inline phpBB/includes/mcp/mcp_reports.php]: 3.1.5-RC1

**Comment by brunoais** [comment]
It shouldn't be failing... Trying again.

File: phpBB/includes/mcp/mcp_reports.php [hunk 2]
```diff
@@ -73,18 +73,47 @@ function main($id, $mode)
...
-				$sql = 'SELECT r.post_id, r.user_id, r.report_id, r.report_closed, report_time, r.report_text, r.reported_post_text, r.reported_post_uid, r.reported_post_bitfield, r.reported_post_enable_magic_url, r.reported_post_enable_smilies, r.reported_post_enable_bbcode, rr.reason_title, rr.reason_description, u.username, u.username_clean, u.user_colour
-					FROM ' . REPORTS_TABLE . ' r, ' . REPORTS_REASONS_TABLE . ' rr, ' . USERS_TABLE . ' u
-					WHERE ' . (($report_id) ? 'r.report_id = ' . $report_id : "r.post_id = $post_id") . '
+					'FROM'		=> array(
+						REPORTS_TABLE			=> 'r',
+						REPORTS_REASONS_TABLE	=> 'rr',
+						USERS_TABLE				=> 'u',
+					),
+
+					'WHERE'		=> '' . (($report_id) ? 'r.report_id = ' . $report_id : "r.post_id = $post_id") . '
```
Comment (marc1706) [inline phpBB/includes/mcp/mcp_reports.php]: Leading ```'' . ``` shouldn't be needed.

Comment (brunoais) [inline phpBB/includes/mcp/mcp_reports.php]: done

**Pull request merged**